### PR TITLE
TRIVIAL: remove the newline character from the CSV header string

### PR DESCRIPTION
**Pull request by kabanoid**
When I call project.upload_multiple and CSV files are processed, the line terminator character remains attached to the name of the last column, causing errors like "Manifest consist of columns that are not in single CSV file". This commit delivers fix for this issue.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/388/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1236/ : SUCCESS in 1m 06s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1506/ : SUCCESS in 3m 57s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/390/ : SUCCESS in 0s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1237/ : SUCCESS in 1m 05s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1507/ : FAILURE in 1m 07s

**Comment by kabanoid** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/391/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1238/ : SUCCESS in 2m 22s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1508/ : FAILURE in 2m 21s

File: lib/gooddata/models/model.rb [hunk 1]
```diff
@@ -188,7 +188,7 @@ def upload_multiple_data(data, project_blueprint, options = { :client => GoodDat
                   end
                 end
               else
-                csv_header = File.open(path, &:gets).split(',')
+                csv_header = File.open(path, &:gets).chomp('').split(',')
```
Comment (kubamahnert) [inline lib/gooddata/models/model.rb]: why is passing an empty string to chomp needed?
"test\n".chomp => "test"
"test\n".chomp '' => "test"

**Comment by kabanoid** [comment]
Hello @kubamahnert , what can we do to get this code merged?

**Comment by kubamahnert** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/397/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1244/ : SUCCESS in 2m 38s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1514/ : FAILURE in 2m 38s

**Comment by kubamahnert** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/401/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1474/ : FAILURE in 14m 58s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul/46/ : FAILURE in 6m 33s

**Comment by kubamahnert** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/402/ : SUCCESS in 29s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1475/ : SUCCESS in 14m 13s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul/47/ : SUCCESS in 52m 11s

**Pull request merged**